docs/reference/offline-stores/redshift.md

```diff
@@ -27,7 +25,39 @@ offline_store:
 ```
 {% endcode %}
 
-Configuration options are available [here](https://github.com/feast-dev/feast/blob/bf557bcb72c7878a16dccb48443bbbe9dc3efa49/sdk/python/feast/infra/offline_stores/redshift.py#L22).
+The full set of configuration options is available in [RedshiftOfflineStoreConfig](https://rtd.feast.dev/en/master/#feast.infra.offline_stores.redshift.RedshiftOfflineStoreConfig).
+
+## Functionality Matrix
```

Comment:
Should we somehow call out that array features don't work?

Reply:
we should, but I think that description belongs to data sources as opposed to offline stores - I can make a ticket for updating the data source docs

Reply:
that's fair. OfflineStore vs DataSource are also a confusing distinction so might have some note on the very top of each offline store that points them to the data source doc